docs/guide/module-hierarchy.rst

```diff
@@ -253,17 +286,28 @@ Spark <http://spark.apache.org/>`__.
    coordinates.
 -  Utilities around creating spark contexts for applications using
    GeoTrellis, including a Kryo registrator that registers most types.
+-  Implements GeoTrellis COGLayers creation, persistence and query mechanisms.
```

Comment:
I think you can remove the `s` from `COGLayers` since `COGLayer` is a singular thing.